### PR TITLE
Updating Sally's suggestions + link

### PR DESCRIPTION
**Pull request by kfklein15**
Signed-off-by: KFK <80272258+kfklein15@users.noreply.github.com>

## What does this change?

Module: Final Projects
Week(s): preparation

## Description

 Changed Sally's suggestion's as well as amended links to team file (which wasn't working anymore) and to the Roles (which have been moved to Gitbook)

## Who needs to know about this?

 @MarianaVazquezCYF @martaesn @ShaunCYFSA @AliciaE08 @caromelendezl 

## Rendered Pages

-----
[View rendered docs/finalproject/prep.md](https://github.com/CodeYourFuture/syllabus/blob/kfklein15-patch-1/docs/finalproject/prep.md)

### CODE REVIEW

File: docs/finalproject/prep.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Getting the Final Projects ready is not a task that can be done a week before th
 
 **When you are on the React Module**
 - [ ] Must review the trainee’s performance and make [readiness for Final Projects decision](https://docs.google.com/document/d/1jMxqI0L7IKFENCQ8Lw-D1lhnj3c3RZI_WK808wG4YJM/edit?usp=sharing)
-- [ ] Identify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
+- [ ] If you haven't yet, make sure you dentify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md]:
```suggestion
- [ ] If you haven't yet, make sure you identify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
```

File: docs/finalproject/prep.md [hunk 2]
```diff
@@ -23,10 +23,10 @@ Getting the Final Projects ready is not a task that can be done a week before th
 - [ ] Let volunteers choose which project they would like to be on
 
 **When you are on SQL Module**
-- [ ] Trainees must have finalised the Full Stack assessment 
+- [ ] Trainees' must have completed up to Level 300 of the Full Stack assessment by week 3 of the module before Final Projects and finalised
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md]:
```suggestion
- [ ] Trainees must have completed up to Level 300 of the [Full Stack Assessment](https://github.com/CodeYourFuture/Table-of-Contents) by week 3 of the module before Final Projects.
```

This is plural, but not plural possessive, in this case. It was the other use of trainees in your previous PR that was plural possessive. It's a minefield! 😅

File: docs/finalproject/prep.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ Getting the Final Projects ready is not a task that can be done a week before th
 
 **When you are on the React Module**
 - [ ] Must review the trainee’s performance and make [readiness for Final Projects decision](https://docs.google.com/document/d/1jMxqI0L7IKFENCQ8Lw-D1lhnj3c3RZI_WK808wG4YJM/edit?usp=sharing)
-- [ ] Identify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
+- [ ] If you haven't yet, make sure you dentify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
 - [ ] Ask internally and externally for possible ideas of charities that could be part of Final Projects. The form used should be [this](https://forms.gle/Gn85Kdna6QB2Qfcp9).
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md]:
```suggestion
- [ ] Ask internally and externally for possible ideas of charities that could be part of Final Projects. The form used should be the [Final Project Proposals form](https://forms.gle/Gn85Kdna6QB2Qfcp9). 
```

Links must make sense out of context. https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html

File: docs/finalproject/prep.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ Getting the Final Projects ready is not a task that can be done a week before th
 
 **When you are on the React Module**
 - [ ] Must review the trainee’s performance and make [readiness for Final Projects decision](https://docs.google.com/document/d/1jMxqI0L7IKFENCQ8Lw-D1lhnj3c3RZI_WK808wG4YJM/edit?usp=sharing)
-- [ ] Identify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
+- [ ] If you haven't yet, make sure you dentify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
 - [ ] Ask internally and externally for possible ideas of charities that could be part of Final Projects. The form used should be [this](https://forms.gle/Gn85Kdna6QB2Qfcp9). 
 - [ ] Start recruiting Tech Lead, Product Managers and UI/UX Designer for Final Projects. See the role descriptions [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles).
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md]:
```suggestion
- [ ] Start recruiting Tech Lead, Product Managers and UI/UX Designer for Final Projects. See the [role descriptions](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles). 
```

Links must make sense out of context. https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html

File: docs/finalproject/prep.md [hunk 5]
```diff
@@ -48,9 +48,9 @@ All code should be done _in public on the trainees' Githubs_, unless there is a
 
 Before class starts volunteer and trainee teams should have been assigned.
 
-You can find a template for organising the teams [here](https://docs.google.com/spreadsheets/d/13WGRmeUZoCZfPV3joO4q69GLIaAiYFnrh9WmZ51FrE8/edit#gid=0)
+You can find a template for organising the teams[here](https://docs.google.com/spreadsheets/d/16vSSJgzCZJKF-2pwuBTkKjJJJ9i1CGRqMbYB-HEO5mo/edit?usp=sharing).
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md]:
```suggestion
Please make a copy of this [template spreadsheet](https://docs.google.com/spreadsheets/d/16vSSJgzCZJKF-2pwuBTkKjJJJ9i1CGRqMbYB-HEO5mo/edit?usp=sharing) to organise the teams and track the progress of the projects.
```

Links must make sense out of context. This is important for accessibility. https://www.w3.org/QA/Tips/noClickHere

File: docs/finalproject/prep.md [hunk 6]
```diff
@@ -48,9 +48,9 @@ All code should be done _in public on the trainees' Githubs_, unless there is a
 
 Before class starts volunteer and trainee teams should have been assigned.
 
-You can find a template for organising the teams [here](https://docs.google.com/spreadsheets/d/13WGRmeUZoCZfPV3joO4q69GLIaAiYFnrh9WmZ51FrE8/edit#gid=0)
+You can find a template for organising the teams[here](https://docs.google.com/spreadsheets/d/16vSSJgzCZJKF-2pwuBTkKjJJJ9i1CGRqMbYB-HEO5mo/edit?usp=sharing). 
 **Repos must be logged on this spreadsheet so everyone can see what is happening during the module.**
-All trainees should make sure that they are comfortable with their roles and what they will be required to do. More information about roles can be found [here](./roles)
+All trainees should make sure that they are comfortable with their roles and what they will be required to do. More information about roles can be found [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles)
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md]:
```suggestion
All trainees should make sure that they are comfortable with their roles and what they will be required to do. More [information about roles](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles).
```

https://www.gov.uk/guidance/content-design/links

File: docs/finalproject/prep.md [hunk 7]
```diff
@@ -23,10 +23,10 @@ Getting the Final Projects ready is not a task that can be done a week before th
 - [ ] Let volunteers choose which project they would like to be on
 
 **When you are on SQL Module**
-- [ ] Trainees must have finalised the Full Stack assessment 
+- [ ] Trainees' must have completed up to Level 300 of the Full Stack assessment by week 3 of the module before Final Projects and finalised 
 - [ ] Must review the trainee’s performance and identify the ones going to Final Projects
 - [ ] Identify trainees that are close in performance and allocate them to the same project. We encourage trainees to work with different people, so try to mix the teams up. Don’t forget to update the spreadsheet.
-- [ ] Have a meeting with all volunteers to go through the details of the final project (presentation to be created with expectation, escalation process, type of support, overview of the trainees per team, etc)
+- [ ] Have a meeting with all volunteers and trainees to go through the details of the final project (presentation to be created with expectation, escalation process, type of support, overview of the trainees per team, etc). Don't forget to record it and share the recording with all the participants.
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md]:
```suggestion
- [ ] Have a meeting with all volunteers and trainees to go through the details of the final project (presentation to be created with expectation, escalation process, type of support, overview of the trainees per team, etc). Don't forget to record it and share the recording with all the participants, with their consent.
```

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
A few tweaks. Mainly around links, and then more SPAG.

**Comment by kfklein15** [comment]
All committed, @SallyMcGrath . Thanks

File: docs/finalproject/prep.md [hunk 8]
```diff
@@ -23,10 +23,10 @@ Getting the Final Projects ready is not a task that can be done a week before th
 - [ ] Let volunteers choose which project they would like to be on
 
 **When you are on SQL Module**
-- [ ] Trainees must have finalised the Full Stack assessment 
+- [ ] Trainees must have completed up to Level 300 of the [Full Stack Assessment](https://github.com/CodeYourFuture/Table-of-Contents) by week 3 of the module before Final Projects.
 - [ ] Must review the trainee’s performance and identify the ones going to Final Projects
```
Comment (SallyMcGrath) [inline docs/finalproject/prep.md]:
```suggestion
- [ ] Must review the trainees' performance and identify the ones going to Final Projects
```

**Review by SallyMcGrath — DISMISSED** [review]
LGTM 🌮

**Review by SallyMcGrath — APPROVED** [review]
and again

**Pull request merged**